frontend/src/PagesAdmin/RecruitmentOverviewPage/Components/RecruitmentStatistics/RecruitmentStatistics.tsx

```diff
@@ -0,0 +1,55 @@
+import styles from './RecruitmentStatistics.module.scss';
+import { Text } from '~/Components/Text/Text';
+import { ReactNode, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { KEY } from '~/i18n/constants';
+
+export function RecruitmentStatistics() {
+  const { t } = useTranslation();
+  const [mockRecruitmentStatOne, setMockRecruitmentStatOne] = useState([
+    { at_date: '28-08-2024', recruitment_applications: 10 },
+    { at_date: '25-08-2024', recruitment_applications: 2 },
+  ]);
+
+  useEffect(() => {
+    //TODO: add dynamic data and might need backend features (in ISSUE #1110)
+    setMockRecruitmentStatOne([
+      {
+        at_date: '28-08-2024---PLACEHOLDER',
+        recruitment_applications: 10,
+      },
+      {
+        at_date: '25-08-2024 ---- PLACEHOLDER',
+        recruitment_applications: 2,
+      },
+    ]);
+  }, []);
```

Comment:
It's fine to use mock data for now, but unnecessary to call `setMockRecruitmentStatOne` with the same init data.

```suggestion
  // TODO: add dynamic data and might need backend features (in ISSUE #1110)
  const [mockRecruitmentStatOne, setMockRecruitmentStatOne] = useState([
    { at_date: '28-08-2024', recruitment_applications: 10 },
    { at_date: '25-08-2024', recruitment_applications: 2 },
  ]);
```

Reply:
done

Reply:
satt mockRecruitmentStatOne som en const